### DISCUSSION TITLE
Error pdn-0175 (pitch is too small)

### DISCUSSION THREAD

**Discussion by krissjarvis** [The-OpenROAD-Project/OpenLane#1783] [Q&A]
[ERROR PDN-0175] Pitch 2.5300 is too small, must be at least 6.6000
Error: pdn_cfg.tcl, 92 PDN-0175
child process exited abnormally

[ERROR]: Creating issue reproducible...

How to change the pitch, or else what should be done in order to get the expected [value(6.6000).
PLEASE REPLY "ASAP"
![image](https://user-images.githubusercontent.com/83774070/235653359-fd63bf44-c39b-4cb0-80f7-ba380ac0465c.png)

**Comment by vijayank88** [comment]
@krissjarvis 
Errors can't fixed/answered in discussion. Open a Github issue with `issue_reproducible` generated by the flow.
From the log I see, you're using old version of OpenLane. 
Update OpenLane and run the flow. If error repeats, follow above steps and create an issue.

**Reply by NiharGowdaS** [reply]
Hey Nihar here.
Did you get any fix for the "pitch is too small, must be atleast 6.600" during Openlane design flow?

**Reply by vijayank88** [reply]
need `issue_reproducible` to provide the solution.

**Reply by NiharGowdaS** [reply]
![Screenshot from 2024-03-22 17-05-45.png](https://github.com/The-OpenROAD-Project/OpenLane/assets/77974814/f4443618-e735-4342-a544-41bf413967f0)

![Screenshot from 2024-03-22 17-06-00.png](https://github.com/The-OpenROAD-Project/OpenLane/assets/77974814/feeca7b6-2f22-4c23-a000-b9fc6e1c0c6e)


I'm changing the FP_PDN_VPITCH value in config.json
But it's not updating in config.tcl during run

**Reply by emillal** [reply]
Had the Same issue. Put FP_PDN_HPITCH also in the json file. It worked for me.

**Comment by jlvsdlv** [comment]
Any update on this? I have the exact same issue : tcl not updating with FP_PDN_HPITCH/VPITCH values set in .json

**Reply by emillal** [reply]
My issue was fixed. See the suggestion made above my be in the comments

**Reply by jlvsdlv** [reply]
Thank you for your response. I unfortunately already tried that (setting both HPITCH and VPITCH in the json) and the tcl is still not picking up those values.
